src/test/java/org/opensearch/neuralsearch/query/HybridQueryAggregationsIT.java

```diff
@@ -496,6 +496,8 @@ private void prepareResourcesForNestegAggregationsScenario(String index) throws
             String ingestBulkPayload = Files.readString(Path.of(classLoader.getResource("processor/ingest_bulk.json").toURI()))
                 .replace("\"{indexname}\"", "\"" + index + "\"");
 
+            // wait until cluster is healthy before running ingest to reduce flakiness
+            waitForClusterHealthGreen(numOfNode, 60, "all");
```

Comment:
Why this test is special that it have to pass "all" while others test does not pass the same value?

Reply:
Your point makes sense, I'll change "all" as default